service/accounts/aws.py

```diff
@@ -54,7 +54,7 @@ def create_identity(self, username, key, secret):
                 identity__provider=self.aws_prov,
                 member__name=username,
                 identity__credential__value__in=[
-                    access_key, secret_key]).distinct()[0]
+                    access_key, secret_key]).distinct()[0]  # noqa=F821
```

Comment:
Since this file is for AWS, I am just suppressing these errors on a line by line basis - we do not expect to use these as we have no plans of supporting AWS.  It might be something that cyverse will want to investigate.